src/modules/create-market/components/create-market-form-define/create-market-form-define.jsx

```diff
@@ -174,7 +175,25 @@ export default class CreateMarketDefine extends Component {
             updateState={this.updateState}
             focused={s.focused}
           />
-          <div className={Styles.CreateMarketDefine_message}>
+          <div
+            className={classNames(
+              Styles.CreateMarketDefine_message,
+              Styles.CutoffMessage
+            )}
+          >
+            <div>
+              No new markets can be created that end after{" "}
+              <span>{CUTOFF_READABLE}</span>. Any markets that end after this
+              date cannot be guaranteed to resolve correctly.{" "}
+              <a
+                href="http://docs.augur.net"
```

Comment:
Is the **Read More** suppose to go http://docs.augur.net or is that just temporary until we have confirmed documentation to provide?

Is it worth adding this as a constant for easy updating later?